Stripe/STPPaymentMethod.m

```diff
@@ -14,7 +14,7 @@
 
 @interface STPPaymentMethod ()
 
-@property (nonatomic, nullable) NSString *identifier;
+@property (nonatomic, nullable) NSString *stripeId;
```

Comment:
I just noticed these properties don't have `strong`/`copy`. I believe the STYLEGUIDE.md wants it to always be included:
https://github.com/stripe/stripe-ios/blob/master/STYLEGUIDE.md#properties

I don't think we usually include `strong`/`copy`/`weak`/`assign` in the `.h` file for `readonly` properties, but I believe it's always included in the `readwrite` re-definition in the `.m` file.

It'd be good to update them all, but I especially think that all of the `NSString` properties in this PR should be `copy`. Faux Pas should catch that, but maybe it doesn't catch it if it's omitted?

Reply:
Ahh thanks, I will make all the properties conform to the styleguide.  

Also agree `copy` should be here, I thought faux pas caught this for me before.